app/src/validators/housing.ts

```diff
@@ -56,7 +56,7 @@ export const housing = Joi.object({
     otherwise: Joi.forbidden()
   }),
   projectName: Joi.string().required().max(255).trim(),
-  projectDescription: Joi.string().max(255).allow(null),
+  projectDescription: Joi.string().max(4000).allow(null),
```

Comment:
just wondering if we decided on 4000? or is it like a db column max character limit?

Reply:
This is what Kyle suggested to me